pkg/mls/store/store.go

```diff
@@ -188,7 +199,16 @@ func (s *Store) PublishIdentityUpdate(ctx context.Context, req *identity.Publish
 		return nil, err
 	}
 
-	return &identity.PublishIdentityUpdateResponse{}, nil
+	result := identityTypes.NewPublishIdentityUpdateResult(
+		req.IdentityUpdate.InboxId,
+		uint64(now),
+		newAccountAddresses,
+		revokedAccountAddresses,
+		[][]byte{}, // TODO: Handle installations added
+		[][]byte{}, // TODO: Handle installations revoked
```

Comment:
What doesn't work without these pieces? Just that they aren't stored?

Reply:
No one is asking for a stream of all the installations added, but I started to stub it out so that we can at least choose to add it later.